addon/components/utils/product-row.hbs

```diff
@@ -0,0 +1,19 @@
+<div class="fx-row fx-gap-px-18 fx-xalign-center padding-right-px-18 product-row {{if @plain "product-row--plain"}}"
```

Comment:
Use single quote inside of double quotes
`class=".... {{if @plain  'product-row--plain'"`